tests/iobeam/test_aifilter.py

```diff
@@ -0,0 +1,94 @@
+from octoprint_mrbeam.iobeam.airfilter import AirFilter
+
+
+def test_model_name():
```

Comment:
I would suggest to split the unit tests that contain more than one assertion in different tests

Reply:
But it is all testing the same functionality, thats why I grouped it

Reply:
But we are testing different scenarios (more than one AAA) so that's why I would separate it, like stated [here](https://www.educative.io/blog/unit-testing-best-practices-overview#:~:text=7.%20One%20use%20case%20per%20unit%20test). But maybe it's just a matter of preference :)